datalake/src/main/java/com/sequenceiq/datalake/flow/dr/restore/DatalakeRestoreFlowConfig.java

```diff
@@ -54,8 +58,7 @@ public class DatalakeRestoreFlowConfig extends AbstractFlowConfiguration<Datalak
                     .from(DATALAKE_DATABASE_RESTORE_START_STATE)
                     .to(DATALAKE_DATABASE_RESTORE_IN_PROGRESS_STATE)
                     .event(DATALAKE_DATABASE_RESTORE_IN_PROGRESS_EVENT)
-                    .failureState(DATALAKE_DATABASE_RESTORE_COULD_NOT_START_STATE)
-                    .failureEvent(DATALAKE_DATABASE_RESTORE_COULD_NOT_START_EVENT)
```

Comment:
@sxxgrc , I have a minor question about this - we are changing this to default failure but we are not doing the same thing for backup - we are still using `DATALAKE_DATABASE_BACKUP_COULD_NOT_START_STATE` and `DATALAKE_DATABASE_BACKUP_COULD_NOT_START_EVENT`, is this on purpose? Also, it seems we can remove `DATALAKE_DATABASE_RESTORE_COULD_NOT_START_EVENT ` in `[DatalakeRestoreEvent.java](https://github.com/hortonworks/cloudbreak/pull/14827/files#diff-b8f8fbc6856f5a50d2bb4d6386764b1d33e6eecec44e145259726f8e4ea6a3ba)`since it's no longer used. Thank you!!

Reply:
No you're correct, this is actually wrong. I was cleaning up places where we can just use the default failure event and seems like I removed this erroneously. Thank you for pointing that out.